include/yaramod/types/expressions.h

```diff
@@ -1199,7 +1242,7 @@ class IdExpression : public Expression
 	 * Constructors
 	 */
 	IdExpression(const std::shared_ptr<Symbol>& symbol)
-		: _symbol(symbol)
+		: _symbol(symbol) //TODO
```

Comment:
What does this TODO mean?

Reply:
Thank you for noticing. This TODO means to check that any IdExpression (or derived class) instance will have its `_symbolToken` valid TokenIt after parsing. I have add test to test this and it seems alright.